docs/guide/search.md

```diff
@@ -251,87 +421,33 @@ ezpublish.search.legacy.gateway.sort_clause_handler.location.depth:
 
 Some examples of common usage:
 ```bash
-# Reindex whole index using parallel process (by default starts by purging the whole index)
+# Reindex the whole index using parallel process (by default starts by purging the whole index)
 # (with the 'auto' option which detects the number of CPU cores -1, default as of 1.13)
 php app/console ezplatform:reindex --processes=auto
 
-# Refresh part of the subtree (implies --no-purge)
+# Refresh a part of the subtree (implies --no-purge)
 php app/console ezplatform:reindex --subtree=2
 
 # Refresh content updated since a date (implies --no-purge)
 php app/console ezplatform:reindex --since=yesterday
 
-# Refresh (or delete when not found) content by IDs  (implies --no-purge)
+# Refresh (or delete when not found) content by IDs (implies --no-purge)
 php app/console ezplatform:reindex --content-ids=3,45,33
 ```
 
-For further info on possible options, see `php app/console ezplatform:reindex --help`.
-
-## Reference
+For further info on possible options, use `php app/console ezplatform:reindex --help`.
 
-See sections on [Search Criteria reference](#search-criteria-reference) and [Sort Clauses reference](#sort-clauses-reference).
- 
 ## Solr Bundle
 
-For use with eZ Publish 5.4, go to [the corresponding documentation page](https://doc.ez.no/display/EZP/Solr+Search+Engine+Bundle) which covers the v1.0 version of the bundle compatible with eZ Publish 5.4.
-
-### What is Solr Search Engine Bundle?
-
-[ezplatform-solr-search-engine](https://github.com/ezsystems/ezplatform-solr-search-engine), as the package is called, aims to be a transparent drop-in replacement for the SQL-based "Legacy" search engine powering eZ Platform Search API by default. When you enable Solr and re-index your content, all your existing Search queries using SearchService will be powered by Solr automatically. This allows you to scale up your eZ Platform installation and be able to continue development locally against SQL engine, and have a test infrastructure, Staging and Prod powered by Solr. This removes considerable load from your database. *See [Architecture](architecture.md)* *for further information on the architecture of eZ Platform.*
+[ezplatform-solr-search-engine](https://github.com/ezsystems/ezplatform-solr-search-engine) aims to be a transparent drop-in replacement for the SQL-based Legacy search engine powering eZ Platform search API by default. When you enable Solr and re-index your content, all your existing Search queries using `SearchService` will be powered by Solr automatically. This allows you to scale up your eZ Platform installation and be able to continue development locally against SQL engine, and have a test infrastructure, Staging and Prod powered by Solr. This removes considerable load from your database. See [further information on the architecture of eZ Platform](architecture.md).
```

Comment:
I disagree about making search API an ordinary word, It's very specific, very eZ "Search API". Besides, at the beginning of this document, the latter version is used anyway.